apps/web/src/pages/Applications/ApplicationProfilePage/ApplicationProfilePage.tsx

```diff
@@ -75,16 +99,194 @@ export const getPageInfo: GetApplicationPageInfo = ({
   };
 };
 
-const ApplicationProfile = ({ application }: ApplicationPageProps) => {
+interface ApplicationProfileProps extends ApplicationPageProps {
+  user: User;
+}
+
+export const ApplicationProfile = ({
```

Comment:
"Just a heads up" does not appear when the profile has incomplete sections
Dropped intentionally? 

![image](https://user-images.githubusercontent.com/40485260/232602864-47b522df-875b-4866-afb2-d80e743ec2d9.png)

Reply:
Not intentionally, just was not part of the acceptance criteria so was missed. Though, this does seem like an important part and I'll work on adding it. Thanks for catching that 🙇‍♀️

Reply:
Added in [add error summary to profile page](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6296/commits/1a6cff5ef47c2d2c5acd0cfe6eea22d0daef8a12)

Reply:
This looks great!